ledger/ledger-api-common/src/main/scala/com/digitalasset/ledger/package.scala

```diff
@@ -10,10 +10,12 @@ package object ledger {
   /** Identifiers used to correlate submission with results. */
   type CommandId = LedgerString
   val CommandId: LedgerString.type = LedgerString
+  val CommandIdKey: String = "daml.command_id"
```

Comment:
This feels like it's slightly the wrong package for tracing related identifiers. Perhaps the metrics package is better? Also fine with these staying here for now and then moving them once more tracing related features land.

Reply:
I second that. The `metrics` library/package seems like a better home for this.

Reply:
Moved to the `metrics` package.